### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by dafortin**
#major#

## CHANGELOG
* **Update** dependencies (ember, prop-types, etc.)
* **Add** `ember-cli-visual-acceptance`
* Run tests on Chrome
* **Add** `.github` folder
* **Add** sass linting
* **Add** `.md` linting
* **Fix** linting issues
* **Add** linting to build

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
+sudo: required
+dist: trusty
```
Comment (sglanzer) [inline .travis.yml]: Have you documented this anywhere (maybe pin to #frost-support?)

File: package.json [hunk 2]
```diff
@@ -28,39 +31,44 @@
...
-    "ember-cli": "~2.6.0",
-    "ember-cli-app-version": "1.0.0",
-    "ember-cli-blanket": "0.9.5",
+    "ember-browserify": "~1.1.0",
+    "ember-cli": "~2.8.0",
+    "ember-cli-app-version": "^1.0.0",
+    "ember-cli-code-coverage": "~0.3.0",
     "ember-cli-dependency-checker": "^1.2.0",
-    "ember-cli-deploy": "1.0.0-beta.1",
+    "ember-cli-deploy": "~0.6.4",
```
Comment (sglanzer) [inline package.json]: I believe ember-cli-deploy + ember-cli-deploy-gh-pages aren't necessary for the github repos

File: tests/dummy/mirage/config.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export default function () {
```
Comment (sglanzer) [inline tests/dummy/mirage/config.js]: I don't see a need for mirage in this repo either

**Comment by sglanzer** [comment]
Approved

File: package.json (hunk 2 above)
Comment (dafortin) [inline package.json]: Agree I think we are using simply the script in /.travis

File: .travis.yml (hunk 1 above)
Comment (dafortin) [inline .travis.yml]: I will :) 👍

**Pull request merged**